l3build-check.lua

```diff
@@ -554,10 +554,19 @@ local function normalize_pdf(content)
   return new_content
 end
 
+function rewrite_log(source, result, engine, errlevels)
+  return rewrite(source, result, normalize_log, engine, errlevels)
+end
+
+function rewrite_pdf(source, result, engine, errlevels)
+  return rewrite(source, result, normalize_pdf, engine, errlevels)
+end
+
```

Comment:
This might look a bit odd because there already is `rewrite` which can be customized with a normalize function, for for other formats it is often easier to access a file directly, therefore this additional indirection.